examples/prediction_upload/geospatial_predictions.ipynb

```diff
@@ -0,0 +1,1563 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    import gdal
</pre><p>Remove this since we are not using it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>